server/service/src/requisition_line/response_requisition_line/delete.rs

```diff
@@ -0,0 +1,184 @@
+use repository::{
+    requisition_row::RequisitionType, RepositoryError, RequisitionLineRowRepository,
+    StorageConnection,
+};
+
+use crate::{
+    requisition::common::{check_approval_status, check_requisition_row_exists},
+    requisition_line::common::check_requisition_line_exists,
+    service_provider::ServiceContext,
+};
+
+#[derive(Debug, PartialEq, Clone, Default)]
+pub struct DeleteResponseRequisitionLine {
+    pub id: String,
+}
+
+#[derive(Debug, PartialEq)]
+
+pub enum DeleteResponseRequisitionLineError {
+    RequisitionLineDoesNotExist,
+    NotThisStoreRequisition,
+    NotAResponseRequisition,
+    RequisitionDoesNotExist,
+    CannotEditRequisition,
```

Comment:
Can you add a `CannotDeleteTransferredRequisition` er or some better naming... This needs to check that the requisition is not created from an Internal Order